packages/client/src/authentication/__helpers__/setupAuthentication.ts

```diff
@@ -0,0 +1,28 @@
+// import { Wallet } from 'ethers';
+
+import { mumbaiSandbox } from '../../consts/environments';
+import { Authentication } from '../Authentication';
+import { setupTestWallet } from './setupTestWallet';
+
+const testConfig = {
+  environment: mumbaiSandbox,
+};
+
+let auth: Authentication;
+
+export async function setupAuthentication(): Promise<Authentication> {
+  if (auth) {
+    return auth;
+  }
+
+  // const wallet = Wallet.createRandom();
+  const wallet = setupTestWallet();
```

Comment:
there is a new ticket to rethink how we do tests in this package https://avara.height.app/lens-sdk/T-6018